.github/workflows/webrtc-draft-release.yml

```diff
@@ -43,6 +43,12 @@ jobs:
     - name: Build
       working-directory: packages/js
       run: npm run build
+    - name: Generate docs
+      working-directory: packages/js
+      run: |
+        npm run docs -- --gitRevision main # IDEA Set git revision to release-it tag
```

Comment:
I added the `IDEA` comment here because I'm not sure how to do this easily with `release-it`. Maybe read the output of `--dry-run`?